src/test/java/nl/tudelft/jpacman/level/PlayerTest.java

```diff
@@ -0,0 +1,76 @@
+package nl.tudelft.jpacman.level;
+
+import static org.junit.Assert.*;
+import nl.tudelft.jpacman.sprite.PacManSprites;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Teste plusieurs méthodes de la classe Player
+ * @author santorin
+ *
+ */
+public class PlayerTest {
+	
+	Player p;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2532399362/issues/source?bid=3256381&fileBranchId=3284358#l16)